salt/_modules/caasp_hosts.py

```diff
@@ -209,8 +209,15 @@ def managed(name=HOSTS_FILE,
     /etc/hosts:
       caasp_hosts.managed
     '''
-    this_roles = __salt__['grains.get']('roles', [])
-    infra_domain = __salt__['caasp_pillar.get'](PILLAR_INTERNAL_INFRA, 'infra.caasp.local')
+    try:
+      this_roles = __salt__['grains.get']('roles', [])
```

Comment:
Indentation seems to be wrong, right? Two spaces here, three later on...

Reply:
it is just a wip stuff for debugging, i agree with your pov :grin: